tests/DynamoDbCompositeModelTest.php

```diff
@@ -150,6 +150,38 @@ public function testStaticMethods()
         ])->toArray());
     }
 
+    public function testConditionContainingCompositeIndexKey()
+    {
+        $fooItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Foo'],
+            'count' => ['N' => 11],
+        ]);
+
+        $barItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Bar'],
+            'count' => ['N' => 9],
+        ]);
+
+        $bazItem = $this->seed([
+            'id' => ['S' => 'id1'],
+            'name' => ['S' => 'Baz'],
+            'count' => ['N' => 10],
+        ]);
+
+        $foundItems = $this->testModel
+            ->where('id', 'id1')
+            ->where('count', '>=', 10) // Test range key support comparison operator other than EQ
+            ->get();
+
+        // If id_count_index is used, $bazItem must be the first found item
```

Comment:
Because items in secondary index will be sorted by range key. And we use `count` as range key here.
```php
'id_count_index' => [
            'hash' => 'id',
            'range' => 'count',
        ],
```